tests/testthat/test-utilities-simulation.R

```diff
@@ -112,3 +112,12 @@ test_that("It returns an emptylist of parameters for a molecule that does not ex
   parameters <- getAllMoleculeParameters("NOPE", sim)
   expect_equal(length(parameters), 0)
 })
+
+context("getAllParametersForSensitivityAnalysisMatching")
+test_that("It returns all parameter potentially interesting for sensitivity analysis for a given wild card path", {
+  sim <- loadTestSimulation("S1", loadFromCache = TRUE)
+  parameters <- getAllParametersForSensitivityAnalysisMatching("**|Volume", sim);
+  varableParameters <- getAllSensitivityAnalysisParametersMatching("**|Volume", sim)
+  expect_gt(length(parameters), length(varableParameters))

```

Comment:
@abdullahhamadeh Here we can see that we get much less parameters using this method than the getAll